content/en/contribute/code/workflow.md

```diff
@@ -28,13 +28,31 @@ A good workflow would be to work locally, pushing to a remote branch as you make
 
 The author and reviewer should use this [guide to code reviewing](https://google.github.io/eng-practices/review/developer/).
 
+#### Suggestions
+
+When doing a code review aim to be extremely clear. This helps things move quickly and avoids lost time in misunderstandings. One especially useful GitHub feature for doing this is suggesting a change. Consider the following example code:
```

Comment:
Should we link the GitHub [related documentation](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request)?

Reply:
> Should we link the GitHub [related documentation](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request)?

Yes! Although, I couldn't find a GitHub source for guidance on this feature specifically. I found a couple pages when it went into beta in 2018 and more general stuff (like the link you mentioned), but I could not find any page from GitHub on this feature. Have you seen anything better @andrablaj?

Reply:
[Step 6](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request) of the link I shared is about the _suggestion_ feature. I didn't find anything more detailed though.

Reply:
The PR can be merged without applying this suggestion, as your documentation is more detailed than the GitHub guide.